SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+The VA.gov team takes security issues in VA.gov seriously. We appreciate your efforts to responsibly 
+disclosure your findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+We release patches for security vulnerabilities. Which versions are eligible receiving such patches depend on the 
+[CVSS v3.0](https://www.first.org/cvss/calculator/3.0) Rating:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| >4.0.x  | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Responsible disclosure security policy
+
+A responsible disclosure policy helps protect users of the project from publicly disclosed security vulnerabilities 
+without a fix by employing a process where vulnerabilities are first triaged in a private manner, and only publicly 
+disclosed after a reasonable time period that allows patching the vulnerability and provides an upgrade path for users.
+
+When contacting us directly via email, we will do our best efforts to respond in a reasonable time to resolve the issue. 
+
+We kindly ask you to refrain from malicious acts that put our users, the project, or any of the project’s team members 
+at risk.
+
+## Reporting a Vulnerability
+
+We consider the security of our systems a top priority. But no matter how much effort we put into system security, 
+there can still be vulnerabilities present.
+
+If you discover a security vulnerability, please report (suspected) security vulnerabilities to 
+**[security@va.gov](mailto:security@va.gov)** and include the word "SECURITY" in the subject line. 
```

Comment:
I don't know anything about this email address. Where does this go? How do we know that this will get to us?

Reply:
I used a made up email address (place holder) since I don't have access to this domain. It'll need to be replaced with a valid email address that goes to the correct team.